understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
                   </p>
                   
                   
-                  <p>When a user is quickly able to find help, they are able to complete the task even
-                     if they encounter challenges.
+                  <p>利用者がすぐにヘルプを見つけることができれば、たとえ困難に直面したとしても作業を完了することができる。
                   </p>
                   
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>メリット</h2>
                <ul>
                   
-                  <li>People who may have difficulty locating help are more likely to find it when it is
-                     consistently located.
+                  <li>ヘルプを見つけるのが困難な人々でも、ヘルプが一貫して置かれるとき、ヘルプを見つける可能性が高くなる。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>On-line job application: Some of the application questions may be hard for new job
-                     seekers to understand even after reading the contextual help. For example, the form
-                     may request their identification number, but they may have several and not know which
-                     one to enter. Consistently located contact information will enable them to use phone
-                     or email so they can get an answer to their question.
+                  <li>オンラインでの求人応募: 応募に関する質問の中には、コンテキストに沿ったヘルプを読んだ後でも、新規求職者にとって理解するのが難しいものがある。例えば、フォームで識別番号を要求するが、複数の識別番号があり、どれを入力すればよいかわからない場合がある。連絡先情報を一貫して配置することで、電話又は電子メールを使用して質問に対する回答を得ることができる。
                   </li>
                   
-                  <li>Medical appointment scheduling form: When the service a patient is trying to book
-                     is not easily findable within the interface, they may need human help. A consistently
-                     located messaging option (chat client) enables them to quickly interact with a staff
-                     person that can help, without requiring them to manage a second interface.
+                  <li>診察予約スケジュールフォーム: 患者が予約しようとしているサービスがインターフェース内で簡単に見つけられない場合、人間の助けが必要になる場合がある。一貫して配置されているメッセージングオプション (チャットクライアント) により、別のインターフェースを管理する必要がなく、サポートできるスタッフと素早く対話できる。
                   </li>
                   
-                  <li>Finding a specific policy or procedure: An employee who needs to complete a work task
-                     may have difficulty locating the specific policy or procedure document on their employer's
-                     Web site. A consistently located "How Do I" page may include the information that
-                     enables them to independently complete this task.
+                  <li>特定のポリシー又は手順を見つける: 仕事の作業を完了する必要がある従業員は、雇用主のウェブサイトで特定のポリシー又は手順の文書を見つけるのが困難な場合がある。一貫して配置されている "How Do I" ページには、このタスクを自力で完了できるようにするための情報が含まれている場合がある。
```

Comment:
"How Do I" を日本語にしてみました。意訳が過ぎるかもですが...

```suggestion
                  <li>特定のポリシー又は手順を見つける: 仕事の作業を完了する必要がある従業員は、雇用主のウェブサイトで特定のポリシー又は手順の文書を見つけるのが困難な場合がある。一貫して配置されている「こんなときはどうする？」ページには、このタスクを自力で完了できるようにするための情報が含まれている場合がある。
```

Reply:
これはなんともですね…こちらも主査レビューに丸投げで…

Reply:
ではここはママ、ということでひとつ…。